containers/daap-athena-load/src/var/task/create_curated_athena_table.py

```diff
@@ -74,191 +257,38 @@ def create_curated_athena_table(
         )
-        raise ValueError(response["QueryExecution"]["Status"].get("StateChangeReason"))
-
-    return query_id
+        return
 
-
-def _get_column_names_and_types(metadata) -> str:
-    select_list = []
-    for column in metadata["TableInput"]["StorageDescriptor"]["Columns"]:
-        col_name = '"' + column["Name"] + '"'
-        col_type = column["Type"] if not column["Type"] == "string" else "VARCHAR"
-        col_no_zero_len_str = f"NULLIF({col_name},'')"
-        select_list.append(f"CAST({col_no_zero_len_str} as {col_type}) as {col_name}")
-
-    select_str = ",".join(select_list)
-
-    return select_str
-
-
-def sql_unload_table_partition(
-    timestamp: str, raw_table: QueryTable, table_path: str, metadata: dict
-) -> str:
-    """
-    generates sql string to unload a timestamped partition
-    of raw data to given s3 location
-    """
-    partition_sql = f"""
-        UNLOAD (
-            SELECT
-                {_get_column_names_and_types(metadata)},
-                '{timestamp}' as extraction_timestamp
-            FROM {raw_table.database}.{raw_table.name}
-        )
-        TO '{table_path}'
-        WITH(
-            format='parquet',
-            compression = 'SNAPPY',
-            partitioned_by=ARRAY['extraction_timestamp']
+    if (
+        s3_client.list_objects_v2(
+            Bucket=data_product_element.curated_data_prefix.bucket,
+            Prefix=data_product_element.curated_data_prefix.key,
+        )["KeyCount"]
+        == 0
+    ):
+        logger.info("This is a new data product.")
+        loader.create_for_new_data_product(
+            raw_data_table=raw_data_table, extraction_timestamp=extraction_timestamp
         )
-    """
-
-    return partition_sql
-
-
-def sql_create_table_partition(
-    curated_table: QueryTable,
-    raw_table: QueryTable,
-    table_path: str,
-    timestamp: str,
-    metadata: dict,
-) -> str:
-    """
-    if the table and data do not exist in curated this
-    will create initial table and parition in glue and file
-    in s3
-    """
-    partition_sql = f"""
-        CREATE TABLE {curated_table.database}.{curated_table.name}
-        WITH(
-            format='parquet',
-            write_compression = 'SNAPPY',
-            external_location='{table_path}',
-            partitioned_by=ARRAY['extraction_timestamp']
-        ) AS
-        SELECT
-            {_get_column_names_and_types(metadata)},
-            '{timestamp}' as extraction_timestamp
-        FROM {raw_table.database}.{raw_table.name}
-    """
-
-    return partition_sql
+        return
 
-
-def refresh_table_partitions(
-    database_name: str, table_name: str, athena_client
-) -> None:
-    """
-    refreshes partitions following an update to a table
-    """
-    athena_client.start_query_execution(
-        QueryString=f"MSCK REPAIR TABLE {database_name}.{table_name}",
-        WorkGroup="data_product_workgroup",
+    logger.info(
```

Comment:
should this be logged as an error?
```suggestion
    logger.error(